### PR TITLE
Add filters as attributes to ElasticSearch

### PR DESCRIPTION
**Pull request by arudnicka**

### CODE REVIEW

**Comment by arudnicka** [comment]
~Need more work - bug with assigned filters to products.~

Resolved.

File: src/utils/product.ts [hunk 1]
```diff
@@ -8,8 +8,8 @@ import { ESProductType, JsonApiDocument, JsonApiResponse, JsonApiSingleResponse
 const productCustomAttributesPrefix = 'prodattr_'
 const productOptionAttributePrefix = 'prodopt_'
 
-const generateOptionAttributeCode = (attributeIdentifier) => `${productOptionAttributePrefix}${attributeIdentifier}`
-const generateCustomAttributeCode = (attributeIdentifier) => `${productCustomAttributesPrefix}${attributeIdentifier}`
+const generateOptionAttributeCode = (attributeIdentifier) => `${attributeIdentifier}`
```
Comment (oleksify) [inline src/utils/product.ts]: @arudnicka Is there a reason for this function to exist? We're taking `attributeIdentifier` and return it. It's not doing anything.

**Review by oleksify — CHANGES_REQUESTED** [review]

File: src/utils/product.ts (hunk 1 above)
Comment (arudnicka) [reply src/utils/product.ts]: Tom wrote them for safety, but I think they are unnecessary. These functions are used many times and I don't want to remove them for now. I will remove them in separate PR, if everything will work properly after testing without them.

File: src/utils/product.ts [hunk 2]
```diff
@@ -8,8 +8,8 @@ import { ESProductType, JsonApiDocument, JsonApiResponse, JsonApiSingleResponse
 const productCustomAttributesPrefix = 'prodattr_'
 const productOptionAttributePrefix = 'prodopt_'
 
-const generateOptionAttributeCode = (attributeIdentifier) => `${productOptionAttributePrefix}${attributeIdentifier}`
-const generateCustomAttributeCode = (attributeIdentifier) => `${productCustomAttributesPrefix}${attributeIdentifier}`
```
Comment (tniezg) [inline src/utils/product.ts]: Can we leave this one and only remove `productOptionAttributePrefix `?

**Review by tniezg — CHANGES_REQUESTED** [review]

**Review by tniezg — APPROVED** [review]

**Pull request merged**